tools/spack/cp2k-dependencies.yaml

```
@@ -9,7 +9,7 @@ spack:
     - "libxsmm @1.17"
     - "fftw @3.3.10 +openmp"
     - "libxc @6.2.2"
-    - "spglib @1.16.2"
+    - "spglib @2.3.1"
```

```suggestion
    - "spglib @2.3.0"
```

@LecrisUT, I haven't yet added `2.3.1` in Spack (will do later today), please use `2.3.0`.

---

The only relevant change there is https://github.com/spglib/spglib/pull/421, but it should not affect this case where the `.mod` files are used. Or rather should not affect at all since the custom fortran wrapper is used

---

Still the same issue though :shrug:

---

Actually, `2.3.0` and `2.2.0` might be too new. I need to fix #3246 before we can update the Spack version (and have the new `spglib` versions available).

---

For reference: https://github.com/spack/spack/pull/43345